bundle/schema/spec.go

```diff
@@ -0,0 +1,210 @@
+package schema
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+type Node struct {
+	Description string `json:"description,omitempty"`
+	Preview     string `json:"x-databricks-preview,omitempty"`
+	Ref         string `json:"$ref,omitempty"`
+
+	// Currently it is only defined for top level schemas
+	JsonPath string `json:"-"`
+}
+
+type Specification struct {
```

Comment:
We don't need the entire type representation on the CLI side yet. Just having `.Components.Schemas` should be enough.

Reply:
Removed 👍